smart-locc.ino

```diff
@@ -37,22 +49,45 @@ void setup() {
     Serial.println("Booted up.");
 }
 
-void loop() {
+void loop()
+{
+    if (!client.connected())
+    {
+        Serial.print("Reconnecting to server...");
+        bool connected = client.connect(CLIENT_ID, USER_TOKEN, PASS);
+        Serial.println(connected ? "success" : "failure");
+    }
+    else
+    {
+        client.loop();
+    }
+    delay(2000);
```

Comment:
This is happening absolutely every iteration, I don't think we want that lag

Reply:
Done